.gitlab/end_to_end.yml

```diff
@@ -45,6 +45,7 @@ end-to-end-foxx-setup:
     - echo "docker run -d \\" >> "${RUN_FILE}"
     - echo "--name \"foxx-${BRANCH_LOWER}-${CI_COMMIT_SHORT_SHA}-${random_string}\" \\" >> "${RUN_FILE}"
     - echo "-e DATAFED_ZEROMQ_SYSTEM_SECRET=\"$CI_DATAFED_ZEROMQ_SYSTEM_SECRET\" \\" >> "${RUN_FILE}"
+    - echo "-e ENABLE_FOXX_TESTS=\"TRUE\" \\" >> "${RUN_FILE}"
```

Comment:
So I understand the context, Foxx tests involve `ArangoDB` (looking at [test_foxx.sh](https://github.com/ORNL/DataFed/blob/9e9beda91bc29409673ca0bfe3f3e07cc4ebc941/core/database/tests/test_foxx.sh#L107))?

Reply:
That's correct, Arangodb has a bit of weird feature that allows you to write JavaScript and run it like database stored procedure, it supports transactions. That's what foxx is.